### DISCUSSION TITLE
module not found error in google collab

### DISCUSSION THREAD

**Discussion by KenDoStudios** [neonbjb/tortoise-tts#353] [Q&A]
ModuleNotFoundError                       Traceback (most recent call last)
[<ipython-input-2-0f6e1f002713>](https://localhost:8080/#) in <module>
      7 import IPython
      8 
----> 9 from tortoise.api import TextToSpeech
     10 from tortoise.utils.audio import load_audio, load_voice, load_voices
     11 

3 frames
[/content/tortoise-tts/tortoise/models/xtransformers.py](https://localhost:8080/#) in <module>
      8 from collections import namedtuple
      9 
---> 10 from einops import rearrange, repeat, reduce
     11 from einops.layers.torch import Rearrange
     12 

ModuleNotFoundError: No module named 'einops'

**Comment by Zavnok** [comment]
[Here](https://github.com/neonbjb/tortoise-tts/issues/347)'s the solution.
